cpp/modmesh/pilot/R3DWidget.hpp

```diff
@@ -69,9 +73,26 @@ class RScene
 
     void setFirstPersonCameraController() { setCameraController(new RFirstPersonCameraController(this)); }
 
+    void updateBoundingBox(QVector3D const & buttom_left, QVector3D const & top_right)
```

Comment:
Used copy by reference for `QVector3D`.

And I found that the same function existed in Qt 5.15 LTS under [`QGeoLocation::setBoundingBox()`](https://doc.qt.io/qt-5/qgeolocation.html#setBoundingBox). However, starting from Qt 6, this was replaced by [`QGeoLocation::setboundingShape()`](https://doc.qt.io/qtforpython-6.5/overviews/qtpositioning-changes-qt6.html#c). 

To prevent conflict with Qt 5.15 LTS, I use `updateBoundingBox()` here.

Reply:
This is good.  I meant to type "pass by reference", but got a typo and wrote "copy by reference".  My bad.  It is "pass by reference".